### DISCUSSION TITLE
Never give up

### DISCUSSION THREAD

**Discussion by dnh33** [maecry/asphyxia-cs2#52] [General]
Stop ruining online video games, thanks

**Comment by maecry** [comment]
> Stop ruining online video games, thanks

Yeah sure also I have clearly stated that this project is for learning purposes for me when I study reverse engineering and game hacking in general. I'm not responsible for other people using this for other purposes and I don't really care about it

**Reply by dnh33** [reply]
ah that's fair. Guess I missed that part. Feel free to remove this issue entirely
